internal/controllers/language_detect/language_detect.go

```diff
@@ -163,19 +166,20 @@ func (ld *LanguageDetect) checkFileExtensionInvalid(path string) bool {
 	return false
 }
 
+// nolint:funlen // method is not necessary broken
 func (ld *LanguageDetect) copyProjectToHorusecFolder(directory string) error {
 	folderDstName := filepath.Join(directory, ".horusec", ld.analysisID.String())
 	err := copy.Copy(directory, folderDstName, ld.filesAndFoldersToIgnore)
 	if err != nil {
 		logger.LogErrorWithLevel(messages.MsgErrorCopyProjectToHorusecAnalysis, err)
-	} else {
-		fmt.Print("\n")
-		logger.LogWarnWithLevel(fmt.Sprintf(messages.MsgWarnMonitorTimeoutIn, ld.configs.TimeoutInSecondsAnalysis))
-		fmt.Print("\n")
-		logger.LogWarnWithLevel(messages.MsgWarnDontRemoveHorusecFolder, folderDstName)
-		fmt.Print("\n")
+		return err
 	}
-	return err
+	fmt.Print("\n")
+	logger.LogWarnWithLevel(fmt.Sprintf(messages.MsgWarnMonitorTimeoutIn, ld.configs.TimeoutInSecondsAnalysis))
+	fmt.Print("\n")
+	logger.LogWarnWithLevel(messages.MsgWarnDontRemoveHorusecFolder, folderDstName)
+	fmt.Print("\n")
```

Comment:
those `fmt.Print("\n")` is really necessary?

Reply:
This code already existed in the previous version I just moved, beleave this is necessary for print in terminal correctly